lib/features/steps/request_assertion_steps.rb

```diff
@@ -63,8 +56,14 @@ def assert_received_requests(request_count, list, list_name)
 # @step_input request_type [String] The type of request (error, session, etc)
```

Comment:
I think it's worth calling out that you can also use "requests" in the docs as it's the only step that works this way. Maybe even have it as a separate step entirely?

```suggestion
# @step_input request_type [String] The type of request (error, session, etc) or "requests" to assert that the test Server hasn't received any requests at all
```

Reply:
Ah, this is an interesting one.  I originally had it as a separate step and Cucumber barfed at it being ambiguous, which makes sense.  I'll add a note to the docs.